request.go

```diff
@@ -0,0 +1,81 @@
+package capnp
+
+import (
+	"context"
+)
+
+type Request struct {
+	method          Method
+	args            Struct
+	client          Client
+	releaseResponse ReleaseFunc
+	future          *Future
+}
+
+func NewRequest(client Client, method Method, argsSize ObjectSize) (*Request, error) {
+	_, seg, err := NewMessage(MultiSegment(nil))
```

Comment:
`NewMultiSegmentMessage(nil)`?  I know we're able to return an error here, but since it should never fail, may as well?

(Alternatively, maybe we want to take an `Arena` as an argument?  Or is that a footgun?)

Reply:
Good catch. Yeah, I'd forgotten that function exists until the other day. Done.